app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -2,6 +2,7 @@ module Katello
   class Api::V2::ContentViewFilterRulesController < Api::V2::ApiController
     before_filter :find_filter
     before_filter :find_rule, :except => [:index, :create]
+    before_filter :create_rules, :only => [:create]
```

Comment:
Does this need to be a before_filter? For me, this just hides logic way up here instead of where you would look by default which is within the `def create` method itself.